Assets/_Scenes/TestScenes/Julian/Scripts/Interaction/PuzzleTerminal.cs

```diff
@@ -91,31 +92,39 @@ void ShowPuzzle()
     public void Lockout(float lockoutTimeInSeconds)
     {
         _unlockTime = Time.time + lockoutTimeInSeconds;
-        _animator.SetInteger("UnlockState", 2);
+        StartCoroutine(LockoutDelay());
     } 
 
     public void Unlock()
     {
         CurrentLockState = LockState.Unlock;
         
         _unlockTime      = 0;
-        _animator.SetInteger("UnlockState", 1);
-        
-        LoadNextScene();
-        if (_PuzzleType == PuzzleType.Frogger && this.gameObject.name == "DoorTerminal")
-        {
-            _nextScene = "EVOS_EndCutscene";
-        }
+        SetAnimationState(CurrentLockState);
+        //
+        // LoadNextScene();
+        // if (_PuzzleType == PuzzleType.Frogger && this.gameObject.name == "DoorTerminal")
+        // {
+        //     _nextScene = "EVOS_EndCutscene";
+        // }
```

Comment:
Because we use the 'Unlock' function in the lockout delay,  this was commented. We should now be using the `LoadSceneTrigger` in it's place now!